### ISSUE TITLE
New project fails to build due to missing SwiftGen template

### ISSUE THREAD

**Issue by colejd** [Rightpoint/ios-template#111]
Building fails with the given output:
```
File written: stitcher/Resources/Generated/UIImage+Asset.swift
File written: stitcher/Resources/Generated/Localized.swift
[31mTemplate named swift4 not found. Use `swiftgen templates list` to list available named templates or use `templatePath` to specify a template by its full path.[0m
Command PhaseScriptExecution failed with a nonzero exit code
```

**Comment by colejd** [comment]
According to the [migration guide](https://github.com/SwiftGen/SwiftGen/blob/master/Documentation/templates/MigrationGuide.md), it looks like the `swift4` template for storyboards was split into `scenes-swift4` and `segues-swift4`.

**Comment by chrisballinger** [comment]
This should be resolved by #119
